cli/cmd/install.go

```diff
@@ -99,7 +144,9 @@ func newCmdInstall() *cobra.Command {
 		Short: "Output Kubernetes configs to install Linkerd",
 		Long:  "Output Kubernetes configs to install Linkerd.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			config, err := validateAndBuildConfig(options)
+			// TODO check with a config already exists in the API and fail if it does.
```

Comment:
When you're generating an issuer, if you're installing over an existing controller, you'll break everything such that existing pods can't communicate with newer pods/control plane instances, since the trust root will change.  We need to make `install` fail if it would likely overwrite an existing install; and we need to introduce a new `upgrade` command that consumes the existing config to generate an updated set of resources (i.e. reusing the existing issuer material).